CREDITS.md

```diff
@@ -1268,6 +1268,8 @@ Fonts
 
 * ["Press Start 2P"](https://www.google.com/fonts/specimen/Press+Start+2P) by 
 [codeman38](mailto:cody@zone38.net) is licensed under [SIL Open Font License, 1.1](http://scripts.sil.org/OFL)
+adapted by JaskRendix, added symbols from DotGothic16 font (https://fonts.google.com/specimen/DotGothic16/)
+licensed under [SIL Open Font License, 1.1](http://scripts.sil.org/OFL)

```

Comment:
@Sanglorian do you think this credit is ok?

Reply:
Link was wrong. I updated. Of course I'm ready to change the format if necessary.